scaling_performance/optimizing_ephemeral_storage.adoc

```diff
@@ -0,0 +1,112 @@
+[[scaling-performance-optimizing-ephemeral-storage]]
+= Optimizing Ephemeral Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+Ephemeral (transient) storage is used by pods for their internal
+operation, such as for temporary files.  The lifetime of this
+ephemeral storage does not extend beyond the life of the individual
+container, and this ephemeral storage cannot be shared across pods.
+
+Prior to {product-title} 3.10, ephemeral local storage was exposed to
+pods via the container’s writable layer, logs directory, and EmptyDir
```

Comment:
we may want to clarify that log management only works if you run crio and use file based logging.  Unless I am mistaken, using docker (with either log driver via json-file or journald) does not support proper accounting of logs.

Reply:
Logging's still available, just not accounted for, I believe.

Reply:
Making this change.